sybil/parsers/rest/codeblock.py

```diff
@@ -34,10 +34,10 @@ class PythonCodeBlockParser(CodeBlockParser):
     A :any:`Parser` for Python :ref:`codeblock-parser` examples.
 
     :param future_imports:
-        An optional list of strings that will be turned into
+        An optional sequence of strings that will be turned into
```

Comment:
I changed this because I wanted it to be in line with the code, and the code defaults to using a tuple.